docs/problem_formulation/problem_formulation_ocp_mex.tex

```diff
@@ -309,7 +312,7 @@ \subsection{Cost module \code{'AUTO'} \label{sec:cost:auto}}
 Set \code{AcadosOcpCost.cost\_type\_0}, \code{AcadosOcpCost.cost\_type}, \code{AcadosOcpCost.cost\_type\_e} to \code{'AUTO'} in order to use this cost module.
 In this case \acados{} detects if the cost function specified is a linear least squares term and transcribes it in the corresponding form.
 Otherwise, it is formulated using the external cost module.
-Note: slack penalties are optional and we plan to detected them from the expressions in future versions.
+Note: slack penalties are optional and we plan to detect them from the expressions in future versions.
```

Comment:
I think this is not planned anymore, and we should remove this sentence altogether.

Reply:
understood, done :)